changelog.md

```diff
@@ -1,3 +1,122 @@
+ <a name="6.4.0"></a>
+# 6.4.0 (2020-09-30)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.3.0...v6.4.0)
+
+We expect this release to ship in the DevTools of [Chrome 88](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## New Contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+
+Denis Seleznev @hcodes
+Irfan Maulana @mazipan
+James Garbutt @43081j
+Matt Hobbs @Nooshu
+Mohammed J. Razem @moerazem
+Pujitha.E @pujitha7
+Quentin @qwright10
+andreizet @andreizet
+
+## Notable Changes
+
+* large-javascript-libraries: move to experimental ([#11484](https://github.com/GoogleChrome/lighthouse/pull/11484))
+
+## New Audits
+
+* unsized-images reland ([#11340](https://github.com/GoogleChrome/lighthouse/pull/11340))
+
+## Core
+
+* tracehouse: add CPU trace profiler model ([#11072](https://github.com/GoogleChrome/lighthouse/pull/11072))
+* stack-packs: add Drupal pack ([#10522](https://github.com/GoogleChrome/lighthouse/pull/10522))
+* stack-packs: move to lighthouse-stack-packs npm package ([#11370](https://github.com/GoogleChrome/lighthouse/pull/11370))
+* axe: use our html snippet rather than axe's ([#11362](https://github.com/GoogleChrome/lighthouse/pull/11362))
+* driver: don't clear indexedb, websql, or localstorage before run ([#11438](https://github.com/GoogleChrome/lighthouse/pull/11438))
```

Comment:
this feels somewhat notable as well, it could potentially be significant to a subset of pages